Mathlib/AlgebraicGeometry/Pullbacks.lean

```diff
@@ -536,6 +536,62 @@ def openCoverOfBase (𝒰 : OpenCover Z) (f : X ⟶ Z) (g : Y ⟶ Z) : OpenCover
       PullbackCone.π_app_right, IsPullback.cone_snd, pullbackSymmetry_hom_comp_fst_assoc]
     rfl
 
+variable (f : X ⟶ Y) (𝒰 : Y.OpenCover) (𝒱 : ∀ i, ((𝒰.pullbackCover f).obj i).OpenCover)
+
+/--
+Given `𝒰 i` covering `Y` and `𝒱 i j` covering `𝒰 i`, this is the open cover
+`𝒱 i j₁ ×_{𝒰 i} 𝒱 i j₂` ranging over all `i`, `j₁`, `j₂`.
```

Comment:
```suggestion
Given `𝒰 i` covering `Y` and `𝒱 i j` covering `𝒰 i`, this is the open cover
`𝒱 i j₁ ×_{𝒰 i} 𝒱 i j₂`  of `X ×_[Y] X` ranging over all `i`, `j₁`, `j₂`.
```

Reply:
Ah, we should agree on one convention of informal fibre product notation. Either latex style or lean style. I slightly prefer the `X ×[Y] Z` version. (this appears below)